Modules/Synchronization/src/Synchronization.Application/Extensions/IEventBusExtensions.cs

```diff
@@ -60,9 +63,13 @@ await Task.WhenAll(new List<Task>
         });
     }
 
+    private static async Task SubscribeToRelationshipTemplatesEvents(IEventBus eventBus)
```

Comment:
These methods are meant to summarize all events belonging to the same module. In other words: you can subscribe from the `SubscribeToRelationshipsEvents` method.